lang/java/compiler/src/test/idl/input/simple.avdl

```diff
@@ -63,6 +60,9 @@ protocol Simple {
     union {null, @foo.foo.bar(42) @foo.foo.foo("3foo") string} prop = null;
   }
 
+  /** An MD5 hash. */
+  fixed MD5(16);
+
```

Comment:
By moving the `MD5` type here, it becomes a forward reference. Without the code changes, this breaks the default value in the `echo` message below.